### PR TITLE
DOC: Edits to attenuation vs extinction

### PR DESCRIPTION
**Pull request by pllim**
See also karllark/dust_extinction#90

### CODE REVIEW

File: docs/dust_attenuation/attenuation.rst [hunk 1]
```diff
@@ -45,7 +48,7 @@ directly proportional to the amount of dust in the system. Hence the ratio of
 attenuations at two different wavelengths *varies* with the amount total system
 dust. In other words, the shape of the attenuation curve varies with the
 total amount of dust in the system unlike the case for extinction.
-This is illustrated below with `WG00` shell, clumpy, mw models where the
+This is illustrated below with ``WG00`` shell, with clumpy ``mw`` models, where the
```
[the remark this answers is not in the document]
Comment (karllark) [reply docs/dust_attenuation/attenuation.rst]: mw should not be quotes.

Fine with WG00 in double quotes.  Thanks for the tip.

File: docs/dust_attenuation/attenuation.rst [hunk 2]
```diff
@@ -99,13 +103,13 @@ plot shows the same curves normalized by Att(V).
...
 
 The impact of the two radiative transfer effects can be illustrated using one of
-the simply system composed of a slab of dust with uniform dust density and two
-stars embedded to varying depths.  These calculations used the DIRTY radiative
+the simple systems composed of a slab of dust with uniform dust density and two
+stars embedded in varying depths.  These calculations used the DIRTY radiative
 transfer model (Gordon et al. 2001; Misselt et al. 2001) and a variant of the
 TRUST slab benchmark geometry (Gordon et al. 2017).  There are 4 different
-geometrical model variants, with the only change being the location of the 1st
+geometrical model variants, with the only change being the location of the 1\ :sup:`st`
```
[the remark this answers is not in the document]
Comment (karllark) [reply docs/dust_attenuation/attenuation.rst]: Great.

File: docs/dust_attenuation/attenuation.rst [hunk 3]
```diff
@@ -135,7 +139,7 @@ understand the radiative effects in different dust-star geometries
 Seon & Draine 2016).  Recently, Law et al. 2018 made publicly available a
 grid of galaxy SEDs from the UV to IR/sub-mm where a stellar population
 synthesis model responsible for the stellar & gas photons is connected to
-the infrared emission through dust grain physics using the DIRTY
```
[the remark this answers is not in the document]
Comment (karllark) [reply docs/dust_attenuation/attenuation.rst]: But not a problem.  Currently your PR removes the whole line.  Did you mean to just change 'infrared' to 'IR'?

[the remark this answers is not in the document]
Comment (pllim) [reply docs/dust_attenuation/attenuation.rst]: @karllark , it does not remove the whole line. I did just change it to "IR". This "red" has a corresponding "green" line just below it. Hope this clarifies the matter.

File: docs/dust_attenuation/attenuation.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (pllim) [reply docs/dust_attenuation/attenuation.rst]: Technically, "mw" is not a proper english word, and probably is acronym for "milky way" and thus strictly speaking should be "MW" (in caps)? Anyway, I undid the change as you requested.

File: docs/dust_attenuation/attenuation.rst [hunk 4]
```diff
@@ -66,6 +69,7 @@ plot shows the same curves normalized by Att(V).
       # defined for normalization
       x_Vband = 0.55
 
+      # TODO: Make plots color-blind friendly?
```
[the remark this answers is not in the document]
Comment (pllim) [reply docs/dust_attenuation/attenuation.rst]: Done in #43

File: docs/dust_attenuation/attenuation.rst [hunk 5]
```diff
@@ -99,13 +103,13 @@ plot shows the same curves normalized by Att(V).
       plt.show()
 
 The impact of the two radiative transfer effects can be illustrated using one of
-the simply system composed of a slab of dust with uniform dust density and two
-stars embedded to varying depths.  These calculations used the DIRTY radiative
+the simple systems composed of a slab of dust with uniform dust density and two
+stars embedded in varying depths.  These calculations used the DIRTY radiative
```
[the remark this answers is not in the document]
Comment (pllim) [reply docs/dust_attenuation/attenuation.rst]: Reverted.

File: docs/dust_attenuation/attenuation.rst [hunk 6]
```diff
@@ -45,7 +48,7 @@ directly proportional to the amount of dust in the system. Hence the ratio of
 attenuations at two different wavelengths *varies* with the amount total system
 dust. In other words, the shape of the attenuation curve varies with the
 total amount of dust in the system unlike the case for extinction.
-This is illustrated below with `WG00` shell, clumpy, mw models where the
+This is illustrated below with ``WG00`` shell, with clumpy mw models, where the
```
[the remark this answers is not in the document]
Comment (pllim) [reply docs/dust_attenuation/attenuation.rst]: Okie dokie.

**Comment by karllark** [comment]
Brilliant.  Thanks!

**Pull request merged**